src/ui/settings/settings_store.ts

```diff
@@ -36,17 +37,63 @@ export const defaultSettings: SettingValues = {
 	consolidateTags: false,
 	uncategorizedVisibility: VisibilityOption.Auto,
 	doneVisibility: VisibilityOption.AlwaysShow,
+	defaultTaskPath: "Tasks.md",
```

Comment:
```suggestion
	defaultTaskPath: '',
```

I don't think we should change the default behaviour for existing users.

Reply:
I was debating this, too. This might be the friction needed to get people over to the easier system. I just went with my instinct on that one. Personally, i don't like setting this again for every new board. What do you personally think - would you use this default path thing?

It was a lot of work to make the non-enforced key system so i may be biased against simply setting a true non-invasive default 😅

Reply:
Obsidian-wide settings might be a solution to this, that preserves the best of both worlds. That way people who want a default path can set it for all new boards at once. But i still prefer onboarding all new users to this default, since it is imo so much more useful.

Perhaps i could build a system that checks if the user is new, and if not, enforce an empty string as default, otherwise Tasks.md.

What do you think?

Reply:
One last option i have thought of, is going through all created kanban boards and checking the ctime. If the newest board would be created before a certain time, the install must be from before a certain cutoff time.

Reply:
Unless i figure something else out, i will opt to add a global settings page and add default task path to that, turned off by default. I will start tracking first installed version, so that after a month or so, the new default can become enforced without breaking existing installations.

Reply:
Could you prompt users to make a choice if not already made? 

eg, `if (defaultTaskPath == null) promptUser()` or something like that.

Unless we tell the user that there is a new setting, it's going to confuse them for their tasks to suddenly start being created in a specific file. Especially since probability would dictate that whatever file name you pick as the default will be different to their preferred file name.